libweb3jsonrpc/Eth.cpp

```diff
@@ -271,6 +271,42 @@ string Eth::eth_sendTransaction(Json::Value const& _json)
 	{
 		throw;
 	}
+	catch (ZeroSignatureTransaction&)
+	{
+		BOOST_THROW_EXCEPTION(JsonRpcException("Zero signature transaction."));
```

Comment:
Let's not use `BOOST_THROW_EXCEPTION` for `JsonRpcException`. I know it's like this in many other places, but it doesn't make sense to me, as `JsonRpcException` is not a boost exception, and `BOOST_THROW_EXCEPTION` will wrap it in boost exception, but this does not improve anything, as it is just caught by libjsonrpc and returned through RPC to the caller. So use just `throw`.